simpleConditional/minTwoNumber.py

```diff
@@ -13,3 +13,5 @@
 # Ex:
 #   minTwoNumber(12, 3) # return 3
 #   minTwoNumber(-2, 3) # return -2
+def minTwoNumber(n1: int, n2: int) -> int:
+    return n2 if n1 > n2 else n1
```

Comment:
Even though this one work, it is hard to read.

”Choose n2 if n1 is greater than n2, otherwise choose n1”. This is hard to read something because you want to “choose” something that not the first things in your expression.

Secondly, the function is about “finding min (smaller)”, does it better to have “return n1 if n1 < n2 else n2”, I can read it as “choose n1 if n1 is less than n2 otherwise n2”.

Remember that code need clarity, it should be easy to read. You can write a bunch of 0/1 and it’s still work. But it not readable.

Reply:
Ok.